src/argus_htmx/themes/utils.py

```diff
@@ -1,21 +1,54 @@
+from importlib.resources import files
+from pathlib import Path
+from re import findall
+
 from django.conf import settings
-from argus_htmx import settings as argus_htmx_settings
+from django.core.exceptions import ImproperlyConfigured
 
+from argus_htmx import settings as fallbacks
 
-def get_themes():
-    return getattr(settings, "DAISYUI_THEMES", argus_htmx_settings.DAISYUI_THEMES)
```

Comment:
tailwind_config depends on this function so we need to keep it (perhaps this one should be `get_themes_from_setting` and the other one `get_theme_names_from_settings`